### PR TITLE
Monitoring & alarms standard

### PR DESCRIPTION
**Pull request by kfriedman**
Hi All,

This is a first draft of the monitoring & alarms standard.

Everyone is welcome/encouraged to comment. I tagged a few people I thought might be interested.

Thanks!

### CODE REVIEW

File: standards/alerting.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
-# Alerting & Notifications
+# Monitoring & Alarms
 
-This is a placeholder for linking until the standard has been drafted.
\ No newline at end of file
+All components MUST have an alarm set up.
```
Comment (rhernand3z) [inline standards/alerting.md]: The word 'components' can have different meanings (e.g. React components, NPM components) should we tailor this to Services such as Lambdas, Back-end Services, API's?

File: standards/alerting.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+Other Alarms
+
+Additional alarms MAY be set up for other metrics. For example: 
+
+- HTTP errors (5xx, 4xx status codes)
+- Lambda errors
+- Kinesis errors
+
+### Alarm Configuration
+
```
Comment (rhernand3z) [inline standards/alerting.md]: Is there a resource you can link to on Alarm Configuration with AWS (that is if developers are using AWS)?

File: standards/alerting.md (hunk 1 above)
Comment (kfriedman) [inline standards/alerting.md]: Good point. Honestly, I wasn't sure what to word to use. A "service" to me is something that serves - generally through an API interface like HTTP, etc. I guess I've been using the word "component" to mean service, consumer, etc. But, you're right, it's not a great word. Maybe I'll just change it to "services, consumers, and applications".

File: standards/alerting.md (hunk 2 above)
Comment (kfriedman) [inline standards/alerting.md]: Good idea. I'll add a link.

**Pull request merged**